src/main/java/my/test/shop/driver/Driver.java

```diff
@@ -0,0 +1,87 @@
+package my.test.shop.driver;
+
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.AbstractDriverOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Driver {
+
+    private static final String BASE_URL = "";
+
+    private final RemoteWebDriver chromeDriver;
+    private final RemoteWebDriver foxDriver;
+
+    private RemoteWebDriver driver;
+    private static Driver instance;
+
+    private Driver() {
+//        driver = new ChromeDriver();
+
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.setPlatformName("macOS 11.00");
+        chromeOptions.setBrowserVersion("75");
```

Comment:
This is hard-code, you need to pass these settings via parameters or/and configuration file (json/txt/any, depends from your implementation)